doc-changelog/action.yml

```diff
@@ -106,44 +156,29 @@ runs:
         # For example, LABELS="enhancement maintenance"
         echo LABELS='"'$pr_labels'"' >> $GITHUB_ENV
 
+    - name: "Set CHANGELOG category based on conventional commit type"
+      if: ${{ inputs.use-labels == 'false' }}
+      shell: python
+      run: |
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
+
+        from parse_pr_title import changelog_category_cc
+
+        cc_type = ${{ env.CC_TYPE }}
+        changelog_category_cc(cc_type)
+
     - name: "Set PR label environment variable"
+      if: ${{ inputs.use-labels == 'true' }}
       shell: python
       run: |
-        import os
-
-        # Create a list of labels found in the pull request
-        # For example, "enhancement maintenance".split() -> ["enhancement", "maintenance"]
-        existing_labels = ${{ env.LABELS }}.split()
-
-        # Dictionary with the key as a label from .github/workflows/label.yml and
-        # value as the corresponding section in the changelog
-        pr_labels = {
-          "enhancement": "added",
-          "bug": "fixed",
-          "dependencies": "dependencies",
-          "maintenance": "changed"
-        }
-
-        def get_changelog_section(pr_labels, existing_labels):
-          """Find the changelog section corresponding to the label in the PR."""
-          label_type = ""
-
-          for key, value in pr_labels.items():
-              if key in existing_labels:
-                  label_type = value
-                  return label_type
-
-          # If no labels are in the PR, it goes into the miscellaneous category
-          label_type = "miscellaneous"
-          return label_type
-
-        # Get the GITHUB_ENV variable
-        github_env = os.getenv('GITHUB_ENV')
-
-        # Append the PR_LABEL with its value to GITHUB_ENV
-        # For example, PR_LABEL="added" if the PR had an "enhancement" label
-        with open(github_env, "a") as f:
-            f.write(f"PR_LABEL={get_changelog_section(pr_labels, existing_labels)}")
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
+
```

Comment:
I really don't like ``sys.path.insert`` but I see no other way around it honestly in this case 😄 - except when we move all these Python scripts to a separate package (see #459)